configure-react.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Configure-React CheatSheet
+description: Package that Help Configure React App with industry standard Faster.
+created: 2022-10-28
+---
+
+## Table of Contents
+
+- [Configure-React CheatSheet for Developers](#configure-react-cheatsheet-for-developers)
```

Comment:
Where is this link pointing?

Reply:
There are 2 external links in the file, 
1st is pointing to the npm directory of that package.
2nd one is pointing to the youtube video that guides the user on how to use the command mentioned in the file.

Other are there to link the table of content only.

Reply:
Above that you are asking is there to link the table of content to the respected content, written in the same file.

Reply:
@onecompiler-ops please let me know what changes is needed, please accept and approve the pull request

Reply:
@onecompiler-ops  But please check commands are working and you can also go through the video for the same https://www.youtube.com/watch?v=2MO1_mCXuds&t=3s

Please support me, by this open-source project of mine can grow, and I will keep improving it more, with time, As the community grows it will help the package also in both ways.